Instructions.txt

```diff
@@ -0,0 +1,132 @@
+########################################
+### Prepare system (Linux Mint 18.1) ###
+########################################
+
+### Step 1: Create Project Directory
+mkdir TKproj
+cd TKproj
+
+
+### Step 2: Clone sources from github
+git clone https://github.com/kevinhughes27/TensorKart.git
+git clone https://github.com/kevinhughes27/mupen64plus-input-bot.git
+git clone https://github.com/mupen64plus/mupen64plus-core.git
+git clone https://github.com/mupen64plus/mupen64plus-input-sdl.git
+
+
+### Step 3: Install dependancies
+sudo apt-get install dpkg-dev build-essential swig python2.7-dev libwebkitgtk-dev libjpeg-dev libtiff-dev checkinstall ubuntu-restricted-extras freeglut3 freeglut3-dev libgtk2.0-dev  libsdl1.2-dev libgstreamer-plugins-base0.10-dev 
+
+sudo apt install python-pip
+sudo pip install --upgrade pip
+sudo pip install -U pip setuptools
+sudo pip install -r requirements.txt
+sudo pip install tensorflow
+
+sudo apt-get install python-tk
+sudo apt-get install xboxdrv
+sudo apt-get install mupen64plus
+
+
+### Step 4: Install m64py GUI for mupen64plus (optional)
+cd ..
+wget "http://sourceforge.net/projects/m64py/files/m64py-0.2.3/m64py_0.2.3-0_all.deb/download" -O m64py_0.2.3-0_all.deb
+
+sudo dpkg -i m64py_0.2.3-0_all.deb
+sudo apt-get -f install
+
+
+### Step 5: Download the Mario Kart ROM
+# DL to TKproj directory
+# https://www.emuparadise.me/Nintendo_64_ROMs/Mario_Kart_64_(USA)/39949-download
+unzip 'Mario Kart 64 (USA).zip'
```

Comment:
I'm not sure its okay to link to this location on GitHub since its probably protected under some copywrite etc.